### ISSUE TITLE
Provide null or undefined children

### ISSUE THREAD

**Issue by voidale** [GeekyAnts/NativeBase#270]
I have hit this bug again, it's pretty much everywhere we need a global function to clear null or undefined children props instead of using `this.props.children` e.g. this is better `this.props.children.filter(child => child)`

`CardItem.js:114` 
````
<Card>
  <CardItem>
    .......
   {isLoading && <Loading />} // error here cant work with null or undefined
  </CardItem>
</Card>
````

This bug happens all around the library.

**Comment by mrpatiwi** [comment]
`FooterTab`has the same issue, when adding a `Button` with a conditional `Badge`.

```js
        <FooterTab>
           <Button {...bprops}>
              {badges > 0 ? (
                <Badge>{badges}</Badge>
              ) : null}
```

It is not filtering the _falsy_ children:
https://github.com/GeekyAnts/NativeBase/blob/master/Components/Widgets/FooterTab.js#L76-L89

**Comment by sankhadeeproy007** [comment]
Hi, we acknowledge it's a major bug and we're working on it. Thanks for pointing it out!

**Comment by sankhadeeproy007** [comment]
Fixed with 0.5.20
